metis-normalization/src/test/java/eu/europeana/normalization/dates/extraction/dateextractors/CenturyDateExtractorTest.java

```diff
@@ -0,0 +1,175 @@
+        Arguments.of("S IV", "03XX", CENTURY_ROMAN),
+        Arguments.of("s V", "04XX", CENTURY_ROMAN),
+        Arguments.of("s VI", "05XX", CENTURY_ROMAN),
+        Arguments.of("s IX", "08XX", CENTURY_ROMAN),
+        Arguments.of("s X", "09XX", CENTURY_ROMAN),
+        Arguments.of("s XI", "10XX", CENTURY_ROMAN),
+        Arguments.of("s XIV", "13XX", CENTURY_ROMAN),
+        Arguments.of("s XV", "14XX", CENTURY_ROMAN),
+        Arguments.of("s XVI", "15XX", CENTURY_ROMAN),
+        Arguments.of("s XIX", "18XX", CENTURY_ROMAN),
+        Arguments.of("s XX", "19XX", CENTURY_ROMAN),
+        Arguments.of("s XXI", "20XX", CENTURY_ROMAN),
+        Arguments.of("s. I", "00XX", CENTURY_ROMAN),
+        Arguments.of("S.I", "00XX", CENTURY_ROMAN),
+        Arguments.of("sec.I", "00XX", CENTURY_ROMAN),
+        Arguments.of("SEC. I", "00XX", CENTURY_ROMAN),
+        Arguments.of("saec.I", "00XX", CENTURY_ROMAN),
+        Arguments.of("SAEC. I", "00XX", CENTURY_ROMAN),
+        Arguments.of("  s I  ", "00XX", CENTURY_ROMAN),
+        Arguments.of("?s. I", "00XX?", CENTURY_ROMAN),
+        Arguments.of("sec. I?", "00XX?", CENTURY_ROMAN),
+        Arguments.of("?saec. I?", "00XX?", CENTURY_ROMAN),
+        Arguments.of("I", "00XX", CENTURY_ROMAN),
+        Arguments.of("IV", "03XX", CENTURY_ROMAN),
+        Arguments.of("V", "04XX", CENTURY_ROMAN),
+        Arguments.of("VI", "05XX", CENTURY_ROMAN),
+        Arguments.of("IX", "08XX", CENTURY_ROMAN),
+        Arguments.of("X", "09XX", CENTURY_ROMAN),
+        Arguments.of("XI", "10XX", CENTURY_ROMAN),
+        Arguments.of("XIV", "13XX", CENTURY_ROMAN),
+        Arguments.of("XV", "14XX", CENTURY_ROMAN),
+        Arguments.of("XVI", "15XX", CENTURY_ROMAN),
+        Arguments.of("XIX", "18XX", CENTURY_ROMAN),
+        Arguments.of("XX", "19XX", CENTURY_ROMAN),
+        Arguments.of("XXI", "20XX", CENTURY_ROMAN),
+        Arguments.of("I", "00XX", CENTURY_ROMAN),
+        Arguments.of("I", "00XX", CENTURY_ROMAN),
+        Arguments.of("I", "00XX", CENTURY_ROMAN),
+        Arguments.of("I", "00XX", CENTURY_ROMAN),
+        Arguments.of("I", "00XX", CENTURY_ROMAN),
+        Arguments.of("I", "00XX", CENTURY_ROMAN),
+        Arguments.of("I", "00XX", CENTURY_ROMAN),
+        Arguments.of("I", "00XX", CENTURY_ROMAN),
+        Arguments.of("  I  ", "00XX", CENTURY_ROMAN),
+        Arguments.of("?I", "00XX?", CENTURY_ROMAN),
+        Arguments.of("I?", "00XX?", CENTURY_ROMAN),
+        Arguments.of("?I?", "00XX?", CENTURY_ROMAN),
+        Arguments.of("saecI", null, null), //Without a dot a space is required
+        Arguments.of("secI", null, null), //Without a dot a space is required
+        Arguments.of("MDCLXX", null, null, null), // Not supported range
+        Arguments.of("IXX", null, null, null), // Invalid roman
+
+        //PATTERN_ROMAN_RANGE
+        Arguments.of("s.I-II", "00XX/01XX", CENTURY_RANGE_ROMAN),
+        Arguments.of("s.II-III", "01XX/02XX", CENTURY_RANGE_ROMAN),
```

Comment:
```Arguments.of("s. I-II", "00XX/01XX", CENTURY_RANGE_ROMAN),``` according to the regex then it will be also valid with a space between the roman range and the s.

Reply:
I added this one too.